src/decision/test/lidar-decision-test.cpp

```diff
@@ -6,10 +6,65 @@
 
 #include <LidarDecision.h>
 #include <gtest/gtest.h>
+#include <vector>
+
+TEST(LidarDecision, obstacle_in_range){
+    sensor_msgs::LaserScan test_scan;
```

Comment:
With gtest, you can run a specific block of code before each test by using a "Setup" function. This can be see [here](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md) (under "Test Fixtures: Using the Same Data Configuration for Multiple Tests"). This would probably help to shorten this code and make your life a lot easier.